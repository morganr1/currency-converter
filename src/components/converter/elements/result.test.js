import React from 'react';
import { shallow } from 'enzyme';
import { Result } from './result.element';

describe('Result element', () => {
  it('renders correctly with the required props passed to it', () => {
    const mockProps = {
      result: 10,
      toCurrency: 'EUR',
      themeType: 'dark'
    };
    const wrapper = shallow(<Result {...mockProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
