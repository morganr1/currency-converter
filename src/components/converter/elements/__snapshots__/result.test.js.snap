// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result element renders correctly with the required props passed to it 1`] = `
<WithStyles(ForwardRef(Typography))
  className="result-text"
  color="textSecondary"
  component="h4"
  style={
    Object {
      "background": "#333333",
    }
  }
  variant="h4"
>
  â‚¬10.00
</WithStyles(ForwardRef(Typography))>
`;
